### ISSUE TITLE
Not compatible with eslint 9+. Invalid `languageOptions.globals` object key "AudioWorkletGlobalScope "

### ISSUE THREAD

**Issue by k-paxian** [vue-a11y/eslint-plugin-vuejs-accessibility#1269]
With latest eslint `9.17.0`
They now validating the configs before applying them
```js
this.language.validateLanguageOptions(this.languageOptions);
```
So it fails to apply this plugin 🤷 

![image](https://github.com/user-attachments/assets/7c2497b7-67c4-48b9-b88d-8fb6e2f5c619)
![image](https://github.com/user-attachments/assets/c82c2af1-5181-4919-95a5-fb6098dc2210)

Also it needs to be aligned with this new way of defining globals
https://eslint.org/docs/latest/use/configure/language-options#predefined-global-variables

**Comment by irmantasluk** [comment]
I have the same issue, same error.

**Comment by Pentadome** [comment]
workaround. We shouldn't need these globals anyway
```ts
import pluginVueA11y from 'eslint-plugin-vuejs-accessibility';

const a11yConfig = pluginVueA11y.configs['flat/recommended'];

a11yConfig.forEach((x) => {
  // there seems to be problem with the globals.
  // We shouldn't need these anyways because we use typescript
  // to define global variables.
  // @ts-expect-error -- deleting is intentional here.
  delete x.languageOptions.globals;
});
```

**Comment by G-Rath** [comment]
The actual fix for this is to update all versions of `globals` in your dependency tree, which you can typically do with `npm update globals` though you might need to "force" a higher major version to be used by explicitly adding `globals` as a dependency and then removing it again.

Specifically, you want to not be using [v13.12.1](https://github.com/sindresorhus/globals/releases/tag/v13.12.1).

An indirect issue with `eslint-plugin-vuejs-accessibility` that would help prevent this is that they're using `globals` as an implicit dependency, which is why package managers don't help you here - this can be fixed by #1280 and/or #1299
